src/operations/hull/hull.js

```diff
@@ -0,0 +1,83 @@
+const flatten = require('../../utils/flatten')
+const areAllShapesTheSameType = require('../../utils/areAllShapesTheSameType')
+
+const { vec2 } = require('../../math')
+
+const { geom2, geom3 } = require('../../geometry')
+
+const hull2DPoints = require('./hull2DPoints')
+
+/*
+ * Create a convex hull of the given geom2 geometries.
+ * @param {...geometries} geometries - list of geom2 geometries
+ * @returns {geom2} new geometry
+ */
+const hullGeom2 = (...geometries) => {
+  geometries = flatten(geometries)
+
+  // extract the unique points from the geometries
+  let uniquepoints = []
+  geometries.forEach((geometry) => {
+    let sides = geom2.toSides(geometry)
+    sides.forEach((side) => {
+      let index = uniquepoints.findIndex((unique) => vec2.equals(unique, side[0]))
+      if (index < 0) uniquepoints.push(side[0])
+    })
+  })
+
+  let hullpoints = hull2DPoints(uniquepoints)
```

Comment:
for coherence I propose using hullPoints2 or heck even hullVec2s ? to match hullGeom2 ?

Reply:
Hmm... but the function works on 2D points. And if you search for 'hull', all documentation about convex hulls says 'set of points'. Even quickhull3D library says 'array of 3D points'.

So, I would favor a generic name so users know this function can be used with generic arrays.

Reply:
I see your point, but then again we have no notion of 'point' just arrays that need to have at least two elements ?

Reply:
OK. I renamed to hullVectors2.

And expect to see hullVectors3 someday in the future. :)